source/renderer/app/containers/MainLayout.js

```diff
@@ -29,21 +29,41 @@ export default class MainLayout extends Component<InjectedContainerProps> {
       environment: { network },
     } = app;
 
-    const sidebarMenus =
+    const nonHardwareWallets =
```

Comment:
@DeeJayElly `nonHardwareWallets` are actually just `wallets`. We don't need to prioritize Hardware Wallets as something main and more important

Reply:
Ok changed.